bench/vectors/README.md

```diff
@@ -1,10 +1,11 @@
-Running the C# program:
-====================================
+# Running the C# program:
```

Comment:
On master, all titles are h1 tags, i.e. they have equal formatting, and presumably equal weight.
Here, the first title is still h1, but all the rest are now h2, i.e. smaller and presumably less important, which is not true, if you read them.

Please restore the old version.

Reply:
Restored :+1: 

Though I think H2s work better when used uniformly (like CONTRIBUTING.md does). Size would still be emphasized when rendered, and they are also bold and underlined. But I missed one h1 in the first commit.

Multiple H1s are often seen as an antipattern (if I remember correctly it was from a screen reader standpoint). But nothing should stop from using them if that's what is desired.

Reply:
I have no objections to multiple h2 tags for all of the current ones, but then we would have to think of a better h1 one. Just promoting the first as h1, does not make sense to me.

Reply:
How about "# Comparison between C# and V implementations of the same algorithm" ?

Reply:
Then the rest can be `## current title`

Reply:
I like the suggestion! What about a little shorter variant of this for a h1?

![Screenshot_20231029_180843](https://github.com/vlang/v/assets/34311583/a8126749-ff75-4174-bf15-453fdfd48cb0)

---

![Screenshot_20231029_180822](https://github.com/vlang/v/assets/34311583/287cf209-3f60-4163-a599-e1c4ceceefc8)

or even shorter:
# C# vs V: Boids Algorithm Benchmark

Or some variant `# C# vs V: Boids Algorithm Comparison`.

Reply:
The second `Comparing C# and V Boids Algorithm Implementations` seems nicer to me, since it avoids `:` and is shorter than the others.